### PR TITLE
Multiple SSR record & more attribute support

### PR DESCRIPTION
**Pull request by creduo**
Related issue #441, for creating reservation,

add support for multiple SSR record
to handle FQTV exception
add carrier & segmentRef attributes

### CODE REVIEW

**Comment by creduo** [comment]
I will follow up for the failed tests.

File: test/Air/transformers/set-segment-ref-for-ssr.test.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+const { expect } = require('chai');
+
+const convert = require('../../../src/Services/Air/transformers/set-segment-ref-for-ssr');
```
Comment (cursedcoder) [inline test/Air/transformers/set-segment-ref-for-ssr.test.js]: would be nice to import this from index module

File: src/Services/Air/transformers/index.js [hunk 2]
```diff
@@ -14,6 +15,7 @@ module.exports = {
   setHasFareBasisFlag,
   setGroupsForSegments,
   addMetaPassengersBooking,
+  setSSRSegmentRef,
```
Comment (cursedcoder) [inline src/Services/Air/transformers/index.js]: I see there is a discrepancy between the file name and method name

File: src/Services/Air/AirParser.js [hunk 3]
```diff
@@ -403,9 +403,12 @@ function airPriceRspPricingSolutionXML(obj) {
     resultXml[root + '_XML'] = lines.join('\n');
   });
 
+  const segs = this.mergeLeafRecursive(objCopy, 'air:AirPriceRsp')['air:AirPriceRsp']['air:AirItinerary']['air:AirSegment'];
```
Comment (cursedcoder) [inline src/Services/Air/AirParser.js]: please use the full name: `segments`

File: test/Air/transformers/set-segment-ref-for-ssr.test.js [hunk 4]
```diff
@@ -0,0 +1,26 @@
+const { expect } = require('chai');
+
+const { setSegmentRefForSSR } = require('../../../src/Services/Air/transformers/index');
```
Comment (cursedcoder) [inline test/Air/transformers/set-segment-ref-for-ssr.test.js]:
```suggestion
const { setSegmentRefForSSR } = require('../../../src/Services/Air/transformers');
```

Comment (creduo) [reply test/Air/transformers/set-segment-ref-for-ssr.test.js]: Oh, my bad, changed as suggested

File: src/Services/Air/AirParser.js (hunk 3 above)
Comment (creduo) [reply src/Services/Air/AirParser.js]: segments was already used at the top of the function, so renamed it to mergedSegments

Comment (cursedcoder) [reply src/Services/Air/AirParser.js]: sure, it is ok!

**Comment by cursedcoder** [comment]
I need approve from @dchertousov before merging this.

**Comment by cursedcoder** [comment]
@creduo thank you!

**Comment by cursedcoder** [comment]
@creduo Btw, you can join our slack as well https://slack.com/share/IRQQNEWGN/Vx3Uu3LW1agxJkqOeNLgMFoX/enQtODc0ODM4NTA2NTY2LWY5YzhhMDRlODQzMzFlNTQ2ODE1NzU3ZTdkOWQxZWJjNDdhMGYxZjdiNmM2ZmY3Yjc5OTQyMWQ1NGI4OWE1ZTI

**Comment by cursedcoder** [comment]
and @shmuga too

**Comment by creduo** [comment]
> @creduo Btw, you can join our slack as well 

Sharing a channel is only possible between two organizations. Maybe what you intended to do was adding a guest account to #uapi-json channel?

**Pull request merged**